[0m20:50:43.319578 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10937ec90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10937e3d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1093a1210>]}


============================== 20:50:43.322320 | b2454351-103e-47ed-b8c1-f642732f14d1 ==============================
[0m20:50:43.322320 [info ] [MainThread]: Running with dbt=1.7.10
[0m20:50:43.322636 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/rahulreddy/.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt init', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m20:50:43.322911 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/Users/rahulreddy/.dbt')}
[0m20:50:43.323084 [info ] [MainThread]: Creating dbt configuration folder at 
[0m20:51:40.174514 [debug] [MainThread]: Starter project path: /Users/rahulreddy/Documents/GitHub/dbt/venv/lib/python3.11/site-packages/dbt/include/starter_project
[0m20:51:40.185964 [info ] [MainThread]: 
Your new dbt project "data_pipeline" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m20:51:40.186520 [info ] [MainThread]: Setting up your profile.
[0m20:54:28.207992 [info ] [MainThread]: Profile data_pipeline written to /Users/rahulreddy/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m20:54:28.218021 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 224.91139, "process_user_time": 0.877392, "process_kernel_time": 0.179765, "process_mem_max_rss": "99549184", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m20:54:28.218812 [debug] [MainThread]: Command `dbt init` succeeded at 20:54:28.218666 after 224.91 seconds
[0m20:54:28.219341 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1093b3b50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1093b3f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105203150>]}
[0m20:54:28.219824 [debug] [MainThread]: Flushing usage events
[0m00:28:39.305926 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109121e50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109171c50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109172650>]}


============================== 00:28:39.308885 | 73b53820-af9a-487c-a44b-59deca403ee8 ==============================
[0m00:28:39.308885 [info ] [MainThread]: Running with dbt=1.7.10
[0m00:28:39.309216 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/rahulreddy/.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'invocation_command': 'dbt deps', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m00:28:39.309705 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/rahulreddy/Documents/GitHub/dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m00:28:39.314764 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_wall_clock_time": 0.038726874, "process_user_time": 0.727198, "process_kernel_time": 0.107524, "process_mem_max_rss": "99254272", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m00:28:39.315108 [debug] [MainThread]: Command `dbt deps` failed at 00:28:39.315037 after 0.04 seconds
[0m00:28:39.315300 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109172550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104296ed0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10918aa50>]}
[0m00:28:39.315509 [debug] [MainThread]: Flushing usage events
[0m00:28:58.957969 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106eb0ed0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106f09510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106f21b50>]}


============================== 00:28:58.960436 | 19d0cda8-1d00-434d-9f80-b47938e52110 ==============================
[0m00:28:58.960436 [info ] [MainThread]: Running with dbt=1.7.10
[0m00:28:58.960753 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/rahulreddy/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt deps', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m00:28:58.961197 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/rahulreddy/Documents/GitHub/dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m00:28:58.962440 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_wall_clock_time": 0.0287405, "process_user_time": 0.726827, "process_kernel_time": 0.095745, "process_mem_max_rss": "98631680", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m00:28:58.962763 [debug] [MainThread]: Command `dbt deps` failed at 00:28:58.962688 after 0.03 seconds
[0m00:28:58.962960 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106f20b90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x102d2b010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x102d2b0d0>]}
[0m00:28:58.963146 [debug] [MainThread]: Flushing usage events
[0m10:56:39.581910 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107a031d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1071d3d10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107a71c90>]}


============================== 10:56:39.584397 | 0efe554c-8c79-492c-8ee7-f015ef5fedd5 ==============================
[0m10:56:39.584397 [info ] [MainThread]: Running with dbt=1.7.10
[0m10:56:39.584709 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/rahulreddy/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt deps', 'send_anonymous_usage_stats': 'True'}
[0m10:56:39.585135 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/rahulreddy/Documents/GitHub/dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m10:56:39.586471 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_wall_clock_time": 0.02928575, "process_user_time": 0.75108, "process_kernel_time": 0.095178, "process_mem_max_rss": "100925440", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m10:56:39.586739 [debug] [MainThread]: Command `dbt deps` failed at 10:56:39.586678 after 0.03 seconds
[0m10:56:39.586918 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107a54910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107a558d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x102ff1bd0>]}
[0m10:56:39.587096 [debug] [MainThread]: Flushing usage events
